spec/core/link.fmf

```diff
@@ -75,6 +75,10 @@ description: |
         This object relates to the provided target.
         This is the **default** relation.
 
+    test-script
+        Web link to the location of the testing script, intended for cases
+        where the tests are stored in a separate repository from the metadata files.
```

Comment:
Is it really intended only for cases when test is in a separate repo? I thought it's also useful for cases when auto generated web link for some reason does not work. And that could be in the same repo. Or not?

Reply:
Thanks for updating the wording.